src/main/resources/static/api.yaml

```diff
@@ -2409,6 +2457,13 @@ components:
           example:
             type: "PAN"
             value: "EEE****00J"
+    LoginTypeResponse:
+      type: object
+      properties:
+        loginType:
+          type: string
+          enum: [CM_ID, OTP]
```

Comment:
Should the value be CM_ID? What you want to convey is login is either by set-password or via a OTP. So, I would think that something like CREDENTIAL, OTP be appropriate?

Reply:
ok I will rename it to loginMode: CREDENTIAL, OTP